File: Job-Portal-BackEnd/src/main/java/lk/ijse/jobportal/service/BussinessFuntionService.java
package lk.ijse.jobportal.service;

import lk.ijse.jobportal.dto.BussinessFuntionDTO;

import java.util.ArrayList;

public interface BussinessFuntionService {

    public boolean addBussinessFuntion(BussinessFuntionDTO bussinessFuntionDTO);

    public ArrayList<BussinessFuntionDTO> getAllBussinessFuntion();
}

File: Job-Portal-FrontEnd/src/app/service/bussiness-funtion.service.ts
import { Injectable } from '@angular/core';
import {HttpClient} from "@angular/common/http";
import {Observable} from "rxjs/index";
import {BussinessFuntion} from "../dto/bussiness-funtion";
import {MAIN_URL} from "./user-service.service";

const URLS="/api/v1/BussinesFuntion"
@Injectable()
export class BussinessFuntionService {

  constructor(private http:HttpClient) { }

  addBussinessFuntion(bussinessFuntion:BussinessFuntion):Observable<boolean>{
    return this.http.post<boolean>(MAIN_URL+URLS,bussinessFuntion);
  }

  getAllBussinessFuntion():Observable<Array<BussinessFuntion>>{
    return this.http.get<Array<BussinessFuntion>>(MAIN_URL+URLS);
  }
}
